pages/user guides/migration_waves.md

```diff
@@ -0,0 +1,72 @@
+---
+toc: false
+title: Migration Waves
+keywords: migration, migrate, cloud, group, move, plan, execute
+last_updated: April, 2025
+summary: "Organize your cloud migration into strategic waves of related resources for phased deployment"
+sidebar: main_sidebar
+folder: userguides
+permalink: migration-waves.html
+---
+
+Migration waves help you organize and execute your cloud migration in manageable, logical groups of related resources. By grouping applications, servers, and database instances that should move together, you can ensure dependencies are maintained and risks are reduced. These strategic waves allow you to break down large migration projects into smaller, coordinated moves that can be planned and executed efficiently.
+
+## Prerequisites
+
+Migration waves exist in projects, and function as a way to break a large project down into logical chunks. In order to proceed, you should first create a project in your workspace. For guidance on how to create a project, see [creating a project](assessment-projects.html#creating-a-project).
+
+## Creating migration waves
+
+To create a migration wave, first navigate to the project where you'd like to create this wave. 
+
+On the project page, you should see a tab called "Migration Waves".
+
+{% include image.html file="migration-waves-1.png" %}
+</br>
```

Comment:
Self closing tags have the `/` at the end :)
```suggestion
<br/>
```

Reply:
oops